### ISSUE TITLE
Need a "default" option for account preference, or save state of last viewed account.

### ISSUE THREAD

**Issue by Jas0n99** [abh/ntppool#184]
Now that a person can have multiple accounts, there doesn't appear to be a way to set a "default" account that is selected / shown on the left sidebar.

There doesn't appear to be any logical sort order for the team names, I'm guessing it's just sorting based on an internal id #.  (Which is okay)

It would be nice if you could set a "default" account (on the account management page) that is chosen when you login / new visit. Or at the very least remember the state of the last chosen account for the user.

**Comment by abh** [comment]
yeah, it looks like it's basically just whatever the database gives back looking for an account you are in: https://github.com/abh/ntppool/blob/master/lib/NTPPool/Control/Manage.pm#L58

I don't particularly want to add an option for this.   Maybe just have the system remember the last account you used?  (Or the last account you used that you are in; so admins default to their own account?).

Another issue you could reasonably open is that the server page loads really slowly if you have a lot of servers in it. 😄  I am guessing that's why you were thinking  of this ...
